src/Snap.cpp

```diff
@@ -317,16 +301,6 @@ SnapResults SnapManager::Snap
    return results;
 }
 
-/* static */ const TranslatableStrings &SnapManager::GetSnapLabels()
```

Comment:
This function might be removed at the earlier commit where SelectionBar stops using it